### ISSUE TITLE
JWT expired should return a 40x error, not a 500.

### ISSUE THREAD

**Issue by mikedevita** [ghaiklor/generator-sails-rest-api#293]
currently if a jwt is expired it returns the following payload with a HTTP 500, i propose it should return 401 or 403, I'd say 403.

```json
{
  "code": "E_INTERNAL_SERVER_ERROR",
  "message": "jwt expired",
  "data": {
    "expiredAt": "2016-11-04T15:36:44.000Z"
  }
}
```

**Comment by mikedevita** [comment]
FYI i patched this manually by editing `isAuthenticated.js` to be 

```javascript
"use strict";

/**
 * isAuthenticated
 * @description :: Policy that inject user in `req` via JSON Web Token
 */

const passport = require('passport');

module.exports = (req, res, next) => {
  passport.authenticate('jwt', (error, user, info) => {

    if (info.name === 'TokenExpiredError') {
      info.status = 401
    }

    if (error || !user) return res.negotiate(error || info);

    req.user = user;

    next();
  })(req, res);
};
```

and adding an if block to check for the status in `responses/negotiate.js`

```
if (statusCode === 401) return res.unauthorized(data, config)
```

**Comment by ghaiklor** [comment]
@mikedevita feel free to create a PR for this :)

**Comment by mikedevita** [comment]
Sure is my method acceptable?

**Comment by ghaiklor** [comment]
@mikedevita looks like this better to place into passport configuration itself, not in a policy:

```js
 if (info.name === 'TokenExpiredError') {
      info.status = 401
    }
```

**Comment by ghaiklor** [comment]
fixed in #300
